operators/index.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="operators-overview"]
+= Operators overview
+include::modules/common-attributes.adoc[]
+:context: operators-overview
+
+toc::[]
+
+[id="importance-of-operators"]
+== The importance of Operators
+include::modules/operators-overview.adoc[leveloffset=+1]
+
+With Operators, you an create applications to monitor the running services in the cluster.
+Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as auto-scaling up and down and backups. All these activities are in a piece of software running inside your cluster.
+To understand more about Operators, see xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[What are Operators?]
+
+[id="developer-tasks"]
+== For developers
+
+As a developer, you can perform the following Operator tasks:
+
+** xref:../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[Create an application from an installed Operator through the web console].
+** xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Install and subscribe an Operator to your namespace].
+** xref:../operators/operator_sdk/osdk-about.adoc#osdk-about[Use Operator SDK to build,test, and deploy an Operator].
+** xref:../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[Install Operator SDK CLI].
```

Comment:
For developers, they should install the `operator-sdk` CLI for developing first, I guess we can list those items with the development steps.